### ISSUE TITLE
Exporter don't run properly

### ISSUE THREAD

**Issue by denisgolius** [guicaulada/vscode-exporter#2]
**Describe the bug**
```
Activation Event:onStartupFinished
Activation Time:23ms
Uncaught Errors (17)
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
Added label "language" is not included in initial labelset: [ 'project', 'folder', 'file', 'type', 'is_untitled' ]
```
I can't see metrics from browser or from curl so they can't be scraped.

```
curl http://localhost:9931/metrics
curl: (7) Failed to connect to localhost port 9931 after 0 ms: Connection refused

curl http://127.0.0.1:9931/metrics
curl: (7) Failed to connect to 127.0.0.1 port 9931 after 0 ms: Connection refused
```

![image](https://user-images.githubusercontent.com/5650611/229984716-eb26f461-878b-4012-89d2-aa3e907aff21.png)

**To Reproduce**

1. Install extension
2. Go to extension page
4. See "RUNTIME STATUS" tab

**Expected behavior**
no issue and showing metrics in browser

**Screenshots**
![image](https://user-images.githubusercontent.com/5650611/229984283-ff1f2374-57f1-427e-98dc-18e7a49bd69e.png)

**Environment details**
```
VSCode
Version: 1.77.0
Commit: 7f329fe6c66b0f86ae1574c2911b681ad5a45d63
Date: 2023-03-29T10:05:35.165Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Linux x64 5.15.0-69-generic
Sandboxed: No
```

![image](https://user-images.githubusercontent.com/5650611/229984408-8c32cee3-02d7-4f9c-ac8d-65fca0c88c5e.png)

**Comment by Bhautik0110** [comment]
The same issue arises! BTW Thanks @guicaulada

**Comment by Josh1866** [comment]
Hi @denisgolius @Bhautik0110 i ran into the same problem as above, do you have any ETA yet for this issue?

**Comment by guicaulada** [comment]
Looking into this now!

**Comment by guicaulada** [comment]
I have fixed the issue with the label on version 6.0.1.

However, regarding your other issue about the metrics not being able to be scraped, you can't see them on your browser or curl because VSCode is not focused and to enable the same port on multiple VSCode windows the web server is only active when the window is focused.

If you want to test the metrics with `curl` you can use it from an embedded terminal within VSCode.
```
curl http://localhost:9931/metrics 
```

Works for me on an embedded terminal.

**Comment by guicaulada** [comment]
Looking into this issue I also found out that while switching windows rapidly the server would fail to start on the new VSCode window because it didn't have time to close on the unfocused window, so I added a retry mechanism so the server is sure to start on the currently focused window even if you switch rapidly between VSCode windows.
